recipe/recipe.yaml

```diff
@@ -37,71 +33,97 @@ source:
         - fix-argminmax.patch
 
 build:
-  number: 0
-  skip: not (match(python, python_min ~ ".*") and is_abi3)
-  python:
-    version_independent: true
+  number: 1
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
+      # Since this meta package is noarch: generic, we only need to build it
+      # for one target platform (does not matter which one)
+      skip: target_platform != 'linux-64'
+      noarch: generic
+      string: py${{ python | version_to_buildstring }}h${{ hash }}_${{ polars_variant | replace('-', '_') }}_${{ build }}
+      variant:
+        down_prioritize_variant: ${{ 0 if polars_variant == 'default' else 1 }}
+    requirements:
+      run:
+        - ${{ pin_subpackage('polars-' ~ polars_variant, lower_bound=version, upper_bound=version) }}
```

Comment:
<img width="382" alt="image" src="https://github.com/user-attachments/assets/86ae2d83-072f-4f10-9711-807474446a84" />

`<1.29.0` is incorrect

Reply:
<img width="511" alt="image" src="https://github.com/user-attachments/assets/8777f476-4d9f-4fbb-aa3d-659e218f24ad" />

I think for all practical purposes this is fine now (as long as we don't start releasing `1.29.0.1`). Now that I think of it though: Why are we using `pin_subpackage` at all now? Shouldn't we be able to just pin `${{ 'polars-' ~ polars_variant }} ${{ version }}` ?

Reply:
not sure, one usually uses `pin_subpackage`. maybe it's related how the DAG is constructed in rattler-build to determine the order of builds? @wolfv do you more details?

Reply:
<img width="525" alt="image" src="https://github.com/user-attachments/assets/5807711e-9db3-4e91-ab7e-236bd23cd2e6" />

seems to work as expected without pin_subpackage.